Update readme to reflect repo (#20)
zowe/zowe-cli-secrets-for-kubernetes: README.md

# Kubernetes Secrets for Zowe CLI and Zowe Explorer

> The Kubernetes Secrets plug-in for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credentials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.

Reviewer: ```suggestion
The Kubernetes Secrets plug-in for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credentials as Kubernetes secrets in your Kubernetes cluster of choice. It changes the default credential manager when installed in Zowe CLI or Zowe Explorer.
```

> This plug-in can also be used as a sample to build your own custom credential manager to use based on your own use case.

Reviewer: ```suggestion
This plug-in also serves as a custom credential manager sample for your own use cases.
```

## How the plug-in works

The plug-in uses the Kubernetes API in order to communicate with your Kubernetes cluster and store secrets on a per user basis.

> In addition, when storing Kubernetes secrets, the secret will be stored in the assigned namespace for the current context or otherwise through an environment variable named `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`.

Reviewer: ```suggestion
In addition, when storing Kubernetes secrets, users can specify a namespace through the environment variables `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`. Otherwise, the secret will be stored in the assigned namespace for the current context.
```

## Requirements

> Before you install and use the plug-in make sure you have the following tools installed:

Reviewer: missing comma between "plug-in" and "make sure"

# Kubernetes Secrets for Zowe CLI & Zowe Explorer

> The Kubernetes Secrets plugin for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credetials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.

Reviewer 2: `credetials` -> `credentials`

> The Kubernetes Secrets plugin for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credetials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.

Reviewer 3: please hyphenate all uses of the word **plug-in** so its consistent w/ Zowe Docs

This plugin can also be used as a sample to build your own custom credential manager to use based on your own use case.

## How the plugin works

The plugin uses the Kubernetes API in order to communicate with your Kubernetes cluster and store secrets on a per user basis.

In addition, when storing Kubernetes secrets, the secret will be stored in the assigned namespace for the current context or otherwise through an environment variable named `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`.

## Requirements

> Before you install and use the plug-in make sure you have the following installed:

Reviewer 3: please define what needs to be installed -- is it "applications"? is it "items"? Rewrite to say: Make sure you have the following applications/items/whatever installed:"

> - Zowe CLI For more information, see [installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).

Reviewer 3: Needs a period between "Zowe CLI" and "For more information"

> - Install [Node.js](https://nodejs.org/en/download) v16.0 or later.

Reviewer 3: - I don't think it's necessary to include "on your computer"
- Remove the verb "install" b/c this is a list of items (according to the sentence in Line 15), not actions.

   The command installs the required dependencies and several development tools. You can run the task at any time to update the tools as needed.

1. To build your code changes, issue the following command:

   ```bash
   npm run build
   ```

   **Note**: When you update package.json to include new dependencies, or when you pull changes that affect package.json, issue the npm update command to download the dependencies.

1. Issue the following command to package the repository and create the `.tgz` file and `.vsix` files to your `/dist` folder.

   ```bash
   npm run package
   ```

1. Issue one of the following commands to install the plug-in:

   ```bash
   zowe plugins install <path to .tgz file>
   ```

Tip: After the installation process completes, it validates that the plug-in was installed correctly and the names of its commands, options, and arguments do not conflict with that of the other plug-ins that you installed into your Zowe CLI implementation.

When the validation process is successful, the following message displays:

```bash
Validation results for plugin '@zowe/secrets-for-kubernetes-for-zowe-cli'
This plugin was successfully validated. Enjoy the plugin.
```

> When an unsuccessful message displays, you can troubleshoot the installation by addressing the issues that the message describes. You can also review the information that is contained in the log file that is located in the Zowe CLI home directory.

Reviewer 3: - is it an "unsuccessful" message or is it a "failure" message?
- I would reword the sentence: " ... by addressing the issues outlined/described in the message."
- Also would tweak the next sentence: "... contained in the log file located in the Zowe CLI ..."

Author: it is a failure message that is displayed when something went wrong when installing the plug-in :)

## Uninstalling

Follow these steps:

1. Issue the following command:

   ```bash

>    zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli

Reviewer 2: Should the plugin name be updated to the K8s plugin?
